From bc17add153bbb287a1991d722aa0631da2290de3 Mon Sep 17 00:00:00 2001
From: knuxify <knuxify@gmail.com>
Date: Fri, 8 Jul 2022 14:06:17 +0200
Subject: [PATCH] Don't re-run plugin tests

There are some expected exceptions in plugin tests which are
ignored in the main test_imports.py file.
---
 test/test_imports.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/test_imports.py b/test/test_imports.py
index 7a69d80..28dea3a 100644
--- a/test/test_imports.py
+++ b/test/test_imports.py
@@ -20,7 +20,8 @@ def load_tests(*_args):
     test_cases = TestSuite()
     home = os.path.dirname(os.path.dirname(__file__))
     for package in pkgutil.walk_packages([f"{home}/blueman"], "blueman."):
-        test_cases.addTest(TestImports(package.name))
+        if not package.name.startswith('blueman.plugins.'):
+            test_cases.addTest(TestImports(package.name))
 
     assert test_cases.countTestCases() > 0
 
-- 
2.37.0

